packages/airnode-protocol/contracts/psp/AirnodePsp.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "../rrp/interfaces/IAirnodeRrp.sol";
+
+contract AirnodePsp {
+    using ECDSA for bytes32;
+
+    struct Subscription {
+        bytes32 templateId;
+        address sponsor;
```

Comment:
Added this after the call because we need a way to match subscription IDs to sponsors (for example see https://github.com/api3dao/airnode/pull/666/files#diff-45d4b511538c5f4fe3ce1dda8891270fe0a6eb927a72603ecceea7d16a7cdbeaR322)